pipeline/metadata/5.2.yaml

```diff
@@ -492,8 +492,24 @@
     - rgw
     - stage-2
 
-- name: "Testing RGW MS ssl ecpool and verifying from primary site"
-  suite: "suites/pacific/rgw/tier-2_rgw_ssl_ecpool_test-ms-verify-io-from-primary.yaml"
+- name: "test-rgw-ssl-ecpool-ms-bucket-objects-gc-policy-on-secondary"
+  suite: "suites/pacific/rgw/tier-2_rgw_ssl_ecpool_ms-bucket-objects-gc-policy-on-secondary.yaml"
+  global-conf: "conf/pacific/rgw/ms-ec-profile-4+2-cluster.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.2"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - sanity
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
```

Comment:
here also we are already having 5 test suite in stage-3 and stage-5, you can refer link https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=rhcsqe&title=Test+Suites 
and can adjust the stage or add new stage (like stage-6 for tier-2) for newly extra added test suite.

Reply:
done